tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -881,6 +881,54 @@ def test_property_metadata_mismatch(self):
         expected_property_2_values = ["", "", "value_1", "value_2", "value_3", "value_4"]
         self.assertListEqual(actual_property_2_values, expected_property_2_values)
 
+    def test_custom_property_order(self):
+        """
+        Test that custom properties are be added to the electrodes table in the order they are set.
+        """
+        self.recording_1.set_property(key="custom_property_x", values=[0.1] * self.num_channels)
+        self.recording_1.set_property(key="custom_property_y", values=[0.2] * self.num_channels)
+        self.recording_1.set_property(key="custom_property_z", values=[0.3] * self.num_channels)
+
+        self.recording_2.set_property(key="custom_property_2", values=["value_1"] * self.num_channels)
+
+        metadata = dict(
+            Ecephys=dict(
+                Electrodes=[
+                    dict(name="custom_property_x", description="custom description."),
+                    dict(name="custom_property_y", description="custom description."),
+                    dict(name="custom_property_z", description="custom description."),
+                    dict(name="custom_property_2", description="custom description."),
+                ]
+            )
+        )
+
+        expected_columns_in_order = [
+            "location",
+            "group",
+            "group_name",
+            "channel_name",
+            "rel_x",
+            "rel_y",
```

Comment:
```suggestion
            "channel_name",
            "group",
            "group_name",
            "location",
            "x",
            "y",
            "z",
            "rel_x",
            "rel_y",
```

The one column we can't control the order of is naturally the `id`, but what do you think about this ordering?

Also could you make up some x/y/z values (possibly using some recent real life examples as a good baseline) to match the location?